### PR TITLE
Make amount of reaction minerals to store a config value

### PR DESCRIPTION
**Pull request by Somotaw**

### CODE REVIEW

**Comment by TooAngel** [comment]
DCBOT: 
This repository is under [democratic collaboration](https://github.com/TooAngel/democratic-collaboration) and will be merged automatically.

The merge decision is based on the outcome of the reviews:
 - `Approve` add the reviewer value (number of commits) to the `metric`
 - `Request changes` substract the reviewer value from the `metric`

 The merge will happen after `(1 - metric/total.votes) * (5 + pull_request.commits * 5))` calculated from the last code change.

Please review the PR to make a good democratic decision.

Summoning some reviewers:
 - @TooAngel: Please look at the code
 - @Nevrdid: A proper review is appreciated
 - @WhiteHalmos: Who are you?

File: src/prototype_room_mineral.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ Room.prototype.reactions = function() {
     //    this.log('Setting reaction: ' + JSON.stringify(this.memory.reaction));
   }
 
-  if (this.terminal.store[this.memory.reaction.result.result] > 1000) {
```
Comment (TooAngel) [inline src/prototype_room_mineral.js]: config value please.

**Comment by TooAngel** [comment]
DCBOT: Adding WIP label, the title is prefixed with [WIP]

**Comment by TooAngel** [comment]
DCBOT: Removing WIP label, the title is not prefixed with [WIP]

File: src/prototype_room_mineral.js (hunk 1 above)
Comment (Somotaw) [inline src/prototype_room_mineral.js]: Done :)

**Comment by TooAngel** [comment]
DCBOT: A new review, yeah.

    Votes: 176/296
    Coefficient: 0.594594594595
    Merging in 2 days -15 hours
    Age 2 days 16 hours

**Pull request merged**